src/components/constants/button-types.ts

```diff
@@ -0,0 +1,5 @@
+export enum ButtonType {
```

Comment:
Nome de enum sempre em maiusculo
```suggestion
export enum BUTTON_TYPE {
```

Reply:
commit: 3a149e0753f3f831d576d78d783a134b912e47ef